frontend/src/components/Home/Home.tsx

```diff
@@ -1,14 +1,17 @@
-import React from 'react';
-import Button from '@mui/material/Button';
-import { AccessAlarm, ThreeDRotation } from '@mui/icons-material';
+import React from "react";
+import Button from "@mui/material/Button";
+import { AccessAlarm, ThreeDRotation } from "@mui/icons-material";
 
-export const Home: React.FC = () => (
-    <>
-        <h1 className="text-yellow-500">This is BanKan</h1>
+const HomeContent: React.FC = () => (
+  <>
+    <h1 className="text-yellow-500">This is BanKan</h1>
 
-        {/* Testing Material UI DELETE WHENEVER */}
-        <Button variant="contained">Cool MUI Button</Button><br/>
-        <AccessAlarm color="primary"/>
-        <ThreeDRotation color="secondary"/>
-    </>
+    {/* Testing Material UI DELETE WHENEVER */}
+    <Button variant="contained">Cool MUI Button</Button>
+    <br />
+    <AccessAlarm color="primary" />
+    <ThreeDRotation color="secondary" />
+  </>
 );
+
+export const Home: React.FC = HomeContent;
```

Comment:
Don't see the point in exporting like this. Also you can leave the Home page as it was, PR #5 should take care of it. (could reduce conflicts)